modules/exploits/windows/mssql/lyris_listmanager_weak_pass.rb

```diff
@@ -35,7 +35,10 @@ def initialize(info = {})
         ],
       'DefaultTarget'  => 0,
       'DisclosureDate' => '2005-12-08'
-      ))
+    ))
+    register_options([
+      OptBool.new('DISPLAY_RESULTS', [true, "Display the Results to the Screen", true])
```

Comment:
Why was this change needed? 👀

Reply:
I think this (and the above) snuck in during a merge, I'll see if I can nuke